client/components/seo/meta-title-editor/mappings.js

```diff
@@ -50,3 +57,13 @@ const tokenToTag = n =>
 
 export const rawToNative = r => removeBlanks( map( split( r, tagPattern ), tagToToken ) );
 export const nativeToRaw = n => join( map( n, tokenToTag ), '' );
+
+export const toApi = compose(
+	partialRight( mapKeys, rearg( snakeCase, 1 ) ), // 1 -> key from ( value, key )
+	partialRight( mapValues, nativeToRaw )          // native objects to raw strings
+);
+
+export const fromApi = compose(
+	partialRight( mapKeys, rearg( camelCase, 1 ) ), // 1 -> key from ( value, key )
+	partialRight( mapValues, rawToNative )          // raw strings to native objects
+);
```

Comment:
@lamosty I sympathize with the fact that many of these functions aren't familiar to many of us, but on the other hand, we have very much adopted **lodash** into the project as a standard library and I don't want to shy away from it just because some of it might be new.

The biggest critique I could agree with is the part about `rearg( …, 1 )`, which was going to be bad no matter how I sliced it. This is due to the way that `mapKeys` works, by sending `( value, key|index, collection )` into the callback. I'm personally opposed to creating variables that we don't end up using because of the confusion it brings in: why is that variable there? why is it named that way? did someone mess up by removing it from the body before but leaving it in? Anyway, even with a closure here like `(value, key) => camelCase( key )` I think the same knowledge about `mapKeys` is needed.

Additionally, `mapKeys`, `mapValues`, and `rearg` are all pretty simple concepts wherein they don't individually do all that much. Hopefully they will be easy to remember and maybe you will find more uses for them in the future.

P.S. The combination of **Dash** and **Alfred** make for an amazing documentation experience. If you haven't looked into them before, it may be worth installing them. I'm only ever a few keystrokes away from documentation on _lodash_, _JavaScript_, _CSS_, _React_, etc…

P.P.S. I added an explanatory comment in f8e681d for why those methods are there.